addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -66,6 +66,16 @@ $bronze: #eea91e;
         p {
             margin-bottom: 0.5rem;
         }
+
+        img {
+            max-width: 100%;
+        }
```

Comment:
we have already a rule to do this
```
    .forum_answer img, .question-block img {
        max-width: 100%
    }
```
but the class `question-block` is no longer used, so the origin of the bug is here
with your fix, you duplicate the rule on forum_answer img...
So I guess you could simply remove the code I pasted here and only keep the rule you added on `o_wforum_readable img`

Reply:
Another fix for 12.0 has been pushed on https://github.com/odoo/odoo/pull/70489